build.sbt

```diff
@@ -92,7 +112,7 @@ val playGenerators = Project("play-grpc-generators", file("play-generators"))
   )
 
 val playTestkit = Project("play-grpc-testkit", file("play-testkit"))
-  .dependsOn(playTestdata % "test")
+  .dependsOn(playTestdata % "test", playActionsTestData % "test")
```

Comment:
helloworld proto in `playActionsTestData` uses a different package from `playTestdata` therefore they can be combined into the same `playTestkit` module.